src/main/java/dk/easv/gui/controllers/AdminViewController.java

```diff
@@ -58,8 +59,14 @@ public void handleLogout() throws IOException {
         this.stage.setTitle("Login");
     }
     @FXML
-    private void handleDelete() {
-
+    private void handleDelete() throws  SQLException{
+        try {
+            Document selectedDocument =  tableView.getSelectionModel().getSelectedValues().get(0);
+            documentModel.deleteDocument(selectedDocument.getId());
+            tableView.getItems().remove(selectedDocument);
```

Comment:
if you reload the observable list in model you dont need to remove the row from table its gonna do it automatically